content/new-web-infrastructure.md

```diff
@@ -0,0 +1,119 @@
+---
+title: New website infrastructure.
+description: >
+ A troubleshooting triage to help you self-diagnose a variety of issues.
+keywords:
+ - support
+ - software
+ - environment
+ - warranty
+ - self-help
+ - tech-support
+ - troubleshooting
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: true
+section: getting-help
+tableOfContents: true
+---
+
+Hello!
+
+You have landed on a place holder page as our web site has had some changes as of Novemember 1st, 2024. Links in support emails that land here are no longer supported by the new site.
+
+We do want to help you if you do have questions, concerns or on going issues! If you are logged into your System76.com account, the following link will allow you to create a support ticket now:
```

Comment:
"on going" should be one word (ongoing).

I'd recommend an Oxford comma after "concerns".